src/components/TextField/textfield.scss

```diff
@@ -0,0 +1,121 @@
+.sb-textfield {
```

Comment:
Could you jump one line between the elements to make more readable?

Reply:
Done!